larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -284,9 +285,11 @@ void OneTimeSetup(TfLiteContext* context, TfLiteNode* node,
 
   // For 'same-zero' padding, compute the padding-correction.
   if (params->padding_type == kTfLitePaddingSame && params->pad_value == 0) {
-    params->padding_buffer.resize(ce::core::PaddingFunctor::get_cache_size(
-        params->filter_height, params->filter_width, params->channels_out,
-        params->dilation_height_factor, params->dilation_width_factor));
+    params->padding_buffer.resize(
+        ce::core::PaddingFunctor::get_cache_size(
+            params->filter_height, params->filter_width, params->channels_out,
+            params->dilation_height_factor, params->dilation_width_factor) +
+        LCE_EXTRA_BYTES);
```

Comment:
I believe that padding is not required for this padding correction tensor or the im2col tensor above because we don't ever read from them beyond bounds, but I've added it to be safe.

I'm also happy to remove these.

Reply:
You're right, its not required for the padding part because this is never done in assembly, only in a separate C++ pass after finishing the bgemm. I'd have a slight preference for removing it to avoid potential confusion on why it's there.